build.rs

```diff
@@ -0,0 +1,14 @@
+// Let people using the standard `cargo x` syntax 
+fn main() {
+    
+    panic!("
+    
+    Enarx uses `cargo-make` to build and run its code:
+    https://github.com/sagiegurari/cargo-make
+
+    You can try `cargo make build` or `cargo make test`.
+
```

Comment:
I don't know where my last comment went :sweat_smile: 

but my suggestion would be to explicitly state here, near the end:

"To install cargo-make, run: cargo install cargo-make"

Reply:
Done!

Reply:
Perhaps something like:
```
To build Enarx:

1. Install cargo-make.
$ cargo install cargo-make

2. Build Enarx.
$ cargo make build

3. You can optionally run tests.
$ cargo make test

For more information, see:
https://github.com/enarx/wiki/...
```